apps/explorer/src/app/components/txs/txs-infinite-list.tsx

```diff
@@ -55,6 +55,9 @@ export const TxsInfiniteList = ({
   error,
   className,
 }: TxsInfiniteListProps) => {
+  const { screenSize } = useScreenDimensions();
+  const stack = ['xs', 'sm', 'md', 'lg'].includes(screenSize);
```

Comment:
suggestion (non-blocking): boolean variable names imo are a lot more understandable when indicated such in the name

```suggestion
  const isStacked = ['xs', 'sm', 'md', 'lg'].includes(screenSize);
```